clouditor-engine-core/src/main/java/io/clouditor/assurance/RuleService.java

```diff
@@ -285,8 +285,13 @@ public void handle(DiscoveryResult result) {
       // evaluate all rules
       rulesForAsset.forEach(
           rule -> {
-            var eval = rule.evaluate(asset);
-
+            EvaluationResult eval;
+            if (!rule.evaluateApplicability(asset)) {
+              // simply add an empty EvaluationResult
+              eval = new EvaluationResult(rule, asset.getProperties());
```

Comment:
Fine for now, although I am not sure what the effect on the UI is with an empty evaluation result